scio-elasticsearch/es8/src/main/scala/com/spotify/scio/elasticsearch/CoderInstances.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.elasticsearch
+
+import co.elastic.clients.elasticsearch.core.bulk.BulkOperation
+import com.spotify.scio.coders._
+
+trait CoderInstances {
+
+  // BulkOperation only has JsonpSerializable, not @JsonpDeserializable
+  // use kryo instead
+  implicit val bulkOperationCoder: Coder[BulkOperation] = Coder.kryo[BulkOperation]
```

Comment:
There is no easy way to deserialize the BulkOperation, fallback with kryo

Reply:
Since the new client API, the model isn't yet in a document form but still a Java object. I think it is not possible get better serialization support, except for the `DeleteOperation` which do not contain document and is annotated with `@JsonpDeserializable`.

The other Operations require to serialize a document, which is done with the internal json mapper from the client or with the provided serializer on the operation (which we should also serialize to reconstruct the operation). I do not know how to do it except by using kryo. Moreover, if a custom serializer is provided, we don't have the associate deserializer to parse the document.

Is it worth to only create a specific case for the `DeleteOperation` ?